acceptance_tests/features/environment.py

```diff
@@ -90,10 +91,7 @@ def after_scenario(context, scenario):
     if "reset_eq_stub" in scenario.tags:
         reset_eq_stub()
 
-    leftover_messages = purge_outbound_topics()
-    if leftover_messages:
-        test_helper.fail(f'There are left over messages on the following subscriptions: {leftover_messages}, see logs '
-                         f'above for details.')
+    purge_outbound_topics()
```

Comment:
I think we should still be failing the after scenario if there are left over messages, so that scenarios aren't knowingly leaving messages unconsumed

Reply:
After some discussion, we've decided that we'll keep an eye on subsequent runs of the ATs to see if it at least helps prevent as many failures on test start-up (mostly on the regression tests). We'll still fail after the scenario though, so will pop this back in 👍